velox/common/base/tests/GTestUtils.h

```diff
@@ -54,6 +54,13 @@
   VELOX_ASSERT_THROW_IMPL(                                     \
       facebook::velox::VeloxRuntimeError, _expression, _errorMessage)
 
+#define VELOX_ASSERT_ERROR_STATUS(_expression, _statusCode, _errorMessage) \
```

Comment:
Add VELOX_ASSERT_ERROR_STATUS if it is needed.

Reply:
CC: @pedroerp